### PR TITLE
Don't create dev folder in C drive

### PR DESCRIPTION
**Pull request by caybokotze**
This should resolve the issue where windows requires a dev folder in the C drive.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -52,12 +52,23 @@ async function checkGitInstalled() {
...
 
 async function checkDirectoryIsGitRepository() {
-  const result = await execAsync(
-    "git rev-parse --is-inside-work-tree 2>/dev/null",
-    {
-      encoding: "utf-8",
-    }
-  );
+  const result = false;
+  if(process.platform === "win32") {
```
Comment (sduduzog) [inline src/app.ts:56]: We might not even need to check platforms here, running this command returns a string "true" if parsing, so looks like we don't need platform specific code anymore

File: src/app.ts [hunk 2]
```diff
@@ -52,12 +52,23 @@ async function checkGitInstalled() {
...
+  } else {
+    result = await execAsync(
+      "git rev-parse --is-inside-work-tree 2>/dev/null",
+      {
+        encoding: "utf-8",
+      }
+    );
+  }
+
   const notAGitRepository = !result.stdout;
```
Comment (sduduzog) [inline src/app.ts:72]: this now can become:
```typescript
const notAGitRepository = result.stdout !== "true"
```

**Review by sduduzog — CHANGES_REQUESTED** [review]

File: src/app.ts (hunk 1 above)
Comment (caybokotze) [reply src/app.ts:56]: Are you saying, take out the check and just have the original code without the ```2>/dev/null``` code?

Comment (sduduzog) [reply src/app.ts:56]: pretty much, but of course with the check if it's the required output, to which I guess the 2>dev/null was there to catch any weird responses and make them a boolean false

File: src/app.ts [hunk 3]
```diff
@@ -52,13 +52,19 @@ async function checkGitInstalled() {
...
 
 async function checkDirectoryIsGitRepository() {
-  const result = await execAsync(
-    "git rev-parse --is-inside-work-tree 2>/dev/null",
-    {
-      encoding: "utf-8",
-    }
-  );
-  const notAGitRepository = !result.stdout;
+  const result = false;
```
Comment (sduduzog) [inline src/app.ts:55]: once a `const` variable is initialized, it cannot be reassigned

**Review by sduduzog — APPROVED** [review]

**Pull request merged**